.gitignore

```diff
@@ -16,3 +16,10 @@ profiles.clj
 dev/user.clj
 .lein-repl-history
 ext/packaging
+Gemfile.lock
+Gemfile.local
+.DS_Store
+.idea/
+/junit/
+/tmp/
+puppet_server.iml
```

Comment:
Got it.  I think I had a global git ignore that I lost when I swapped laptops recently.  I'll remove the non-project specific ignores.

The /tmp/ and /junit/ directories are project-specific in that they're created and fill up after running beaker repetitively.